### PR TITLE
chore: updates kommander on-premise installation tutorial.

### PR DESCRIPTION
**Pull request by tillt**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.d2iq.com/browse/D2IQ-69966

## Description of changes being made
Updates on-premise installation tutorial:
- updates button titles
- fixes embedded HTML styling
- adds a crucial preparation step
- fixes linked image title

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 1]
```diff
@@ -61,10 +61,15 @@ all:
 
 Once you have both `cluster.yaml` file and `inventory.yaml` file, you need to add a provisioner for your provider. Follow these steps:
 
-1. Navigate to **Administration / Cloud Providers**.
-2. Select the **Add Provider** button.
-3. Select **On Premise**.
-4. Enter a name for your provider and insert the full contents of your private ssh key, then hit **Verify** and **Save**.
+Make sure you have selected a workspace - for example the "Default Workspace".
+
+- Navigate to **Administration / Infrastructure Providers**.
```
Comment (armandgrillet) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:66]: The previous list style (1. 2. 3. 4.) seems better to me in the context of sequential steps and I don't think the empty lines are necessary.

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 2]
```diff
@@ -74,17 +79,17 @@ You can now launch your cluster.
...
 
 Select **Upload YAML** to create a cluster.
 
 ![Upload YAML](/ksphere/kommander/1.1/img/add-cluster.png)
 
 #### Fill out the form
 
-1. Enter a unique name for your cluster.
-2. At the top of the `cluster.yaml` text field, fill the following information:
+- Enter a unique name for your cluster.
```
Comment (armandgrillet) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:90]: Longer steps here thus it makes sense to change them as you did 👍 .

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 3]
```diff
@@ -97,10 +102,9 @@ spec:
 ---
 ```
 
-<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above into the **cluster.yaml** field before you insert your `cluster.yaml`. For user, it is the ssh user tied to the keys pushed to your vms. This is the same user tied to the key uploaded in the cloud provider step.
-</p>
+<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above <i>ClusterProvisioner</i> into the <i>cluster.yaml</i> field and then add the contents of your <code>cluster.yaml</code> after the three dashes. Replace <i>&lt;username&gt;</i> with the SSH user tied to the keys pushed to your machines. This is the same user tied to the key uploaded in the cloud provider step.</p>
```
Comment (armandgrillet) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:105]: The use of italics for `ClusterProvisioner` and `cluster.yaml` seems non-conventional, why not keeping it as it was?

Comment (tillt) [reply pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:105]: This is HTML - it is broken when leaving as is.

Screenshot 2020-07-23 at 14 37 16

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md (hunk 1 above)
Comment (tillt) [reply pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:66]: It did not fully work the way it was due to the embedded HTML. Note how the numbers jump around.

Screenshot 2020-07-23 at 14 38 45

Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:66]:
```suggestion
- Go to **Administration / Infrastructure Providers**.
```

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 4]
```diff
@@ -61,10 +61,15 @@ all:
...
-4. Enter a name for your provider and insert the full contents of your private ssh key, then hit **Verify** and **Save**.
+Make sure you have selected a workspace - for example the "Default Workspace".
+
+- Navigate to **Administration / Infrastructure Providers**.
+
+- Select the **Add Infrastructure Provider** button.
+
+- Select **On Premise**.
+
+- Enter a name for your provider and insert the full contents of your private ssh key, then hit **Verify** and **Save**.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:72]:
```suggestion
- Enter a name for your provider and insert the full contents of your private ssh key, then select **Verify** and **Save**.
```

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 5]
```diff
@@ -74,17 +79,17 @@ You can now launch your cluster.
...
 
 ![Upload YAML](/ksphere/kommander/1.1/img/add-cluster.png)
 
 #### Fill out the form
 
-1. Enter a unique name for your cluster.
-2. At the top of the `cluster.yaml` text field, fill the following information:
+- Enter a unique name for your cluster.
 
+- At the top of the `cluster.yaml` text field, fill the following information:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:92]:
```suggestion
- At the top of the `cluster.yaml` text field, enter the following information:
```

File: pages/ksphere/kommander/1.1/tutorials/on-prem/index.md [hunk 6]
```diff
@@ -97,10 +102,9 @@ spec:
 ---
 ```
 
-<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above into the **cluster.yaml** field before you insert your `cluster.yaml`. For user, it is the ssh user tied to the keys pushed to your vms. This is the same user tied to the key uploaded in the cloud provider step.
-</p>
+<p class="message--important"><strong>IMPORTANT: </strong>You must paste the above <i>ClusterProvisioner</i> into the <i>cluster.yaml</i> field and then add the contents of your <code>cluster.yaml</code> after the three dashes. Replace <i>&lt;username&gt;</i> with the SSH user tied to the keys pushed to your machines. This is the same user tied to the key uploaded in the cloud provider step.</p>
 
-4. Make sure the Kommander addon is disabled with `enabled: false`.
+- Make sure the Kommander addon is disabled with `enabled: false`.
```
Comment (russurquhart1) [inline pages/ksphere/kommander/1.1/tutorials/on-prem/index.md:107]:
```suggestion
- Ensure the Kommander addon is disabled with `enabled: false`.
```

**Pull request merged**